### ISSUE TITLE
Problems with finetuned flow used in CosyVoice2 while in SFT scenario

### ISSUE THREAD

**Issue by OswaldoBornemann** [FunAudioLLM/CosyVoice#826]
So we tried to finetune flow module used in CosyVoice2 in the SFT scenario. We found that in the inference, the generated speech quality will be better if we feed the prompt speech into the flow model as input as well, while the fine-tuned flow module generated poor speech quality with zeros prompt speech. Is that normal? @aluminumbox

**Comment by aluminumbox** [comment]
we haven't update cosyvoice2 finetune script yet, you can finetune it, but you need to add dynamic chunk mask by yourself

**Comment by OswaldoBornemann** [comment]
@aluminumbox Yes I see your point. My point is that when I finetune it without using dynamic mask, the quality of speech produced by the process is better with prompt speech than without prompt speech, and I am in the SFT scenario.

**Comment by ukemamaster** [comment]
> we haven't update cosyvoice2 finetune script yet, you can finetune it, but you need to add dynamic chunk mask by yourself

When do you plan to update the finetune script for CosyVoice2 model?
